docs/sites.yml

```diff
@@ -2254,7 +2254,15 @@
   url: 'https://500tech.com/'
   featured: false
   categories:
-    - Web Dev
+    - Web Devhttps://github.com/amit-deligence/gatsby.git
```

Comment:
I don't think that is right :)

Reply:
Sorry by mistake I change in this line. Now I have made the changes please review it.